### PR TITLE
[5.0] Use the registry in the banners clients template file

### PR DESCRIPTION
**Pull request by laoneo**
### Summary of Changes
Use the registry instead of CMSObject in the banners clients layout file.

### Testing Instructions
- Create a banners client
- Open the clients list

### Actual result BEFORE applying this Pull Request
All works.

### Expected result AFTER applying this Pull Request
All works.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: administrator/components/com_banners/tmpl/clients/default.php [hunk 1]
```diff
@@ -35,7 +35,7 @@
 $userId     = $user->get('id');
 $listOrder  = $this->escape($this->state->get('list.ordering'));
 $listDirn   = $this->escape($this->state->get('list.direction'));
-$params     = $this->state->params ?? new CMSObject();
+$params     = $this->state->get('params', new Registry());
```
Comment (HLeithner) [inline administrator/components/com_banners/tmpl/clients/default.php]: in this case new Registry will be called reg less if needed or not, should we really do this?

Comment (laoneo) [reply administrator/components/com_banners/tmpl/clients/default.php]: I can revert, just thought it is more clear like that.

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on d9ad4deafdfed252c83a7b8dae6ddf900d7891e4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41062">issues.joomla.org/tracker/joomla-cms/41062</a>.</sub>

File: administrator/components/com_banners/tmpl/clients/default.php (hunk 1 above)
Comment (HLeithner) [reply administrator/components/com_banners/tmpl/clients/default.php]: It is more clear but I think performance wins here. thanks

**Comment by HLeithner** [comment]
thanks

**Pull request merged**